A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -853,16 +853,23 @@ class A320_Neo_MFD_NDInfo extends NavSystemElement {
         }
         const ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
         const groundSpeed = Math.round(Simplane.getGroundSpeed());
+        const trueAirSpeed = SimVar.GetSimVarValue("AIRSPEED TRUE", "knots");
+        const windStrength = SimVar.GetSimVarValue("AMBIENT WIND VELOCITY", "knots");
         const gs = this.ndInfo.querySelector("#GS_Value");
         const tas = this.ndInfo.querySelector("#TAS_Value");
         const wd = this.ndInfo.querySelector("#Wind_Direction");
         const ws = this.ndInfo.querySelector("#Wind_Strength");
         const wa = this.ndInfo.querySelector("#Wind_Arrow");
         const wptg = this.ndInfo.querySelector("#Waypoint_Group");
         const wptPPOS = this.ndInfo.querySelector("#Waypoint_PPOS");
-        if (ADIRSState != 2 || groundSpeed <= 100) {
-            //Hide TAS, and wind info
+        if (ADIRSState != 2 || trueAirSpeed < 60) {
+            //TAS info Conditions
             tas.textContent = "---";
+        } else {
+
+        }
```

Comment:
I'm unable to review the correctness of this PR, as I don't know enough about the airplane in this area to determine that. So just from a code point of view:
**suggestion**: remove the `else` branch.

Reply:
Will remove the else :) according to komp it should use 60kts cas instead of tas for the tas to show up but i couldnt find a simvar for it so i took tas which behaves almost the same